src/strings-in-block/class-base.php

```diff
@@ -27,9 +27,16 @@ protected function get_block_config( \WP_Block_Parser_Block $block, $type ) {
 
 		if ( isset( $block->blockName, $this->block_types[ $block->blockName ][ $type ] ) ) {
 			return $this->block_types[ $block->blockName ][ $type ];
-		} else {
-			return null;
+		} elseif ( isset( $block->blockName) ) {
+			$block_name_arr  = explode( '/', $block->blockName );
+			$block_namespace = reset( $block_name_arr );
+
+			if ( isset( $this->block_types[ $block_namespace ][ $type ] ) ) {
+				return $this->block_types[ $block_namespace ][ $type ];
+			}
 		}
+
+		return null;
```

Comment:
@strategio can't you return an empty array so you always return one type of data?

Reply:
Yes, actually I was already returning an empty array on the new class `Attributes::getKeyConfig` but I didn't change it on the existing one `HTML`.

That's done now.

Reply:
Eventually, I'll keep the existing code because it causes a test to fail on the `HTML` side (in `HTML::find` where we check `is_array`). I am not sure the test is correct but I prefer not to change it now.